examples/3d/vertex_colors.rs

```diff
@@ -33,6 +33,8 @@ fn setup(
             .collect();
         colorful_cube.insert_attribute(Mesh::ATTRIBUTE_COLOR, colors);
     }
+    // Remove unused texture coordinates
+    colorful_cube.remove_attribute(Mesh::ATTRIBUTE_UV_0);
```

Comment:
I feel like this is unnecessary as it has no noticeable impact on the example.

Reply:
I did it because I remember someone saying on Discord that Bevy's examples served double-duty both as examples and kind-of rendering feature-coverage tests. Technically, removing the attribute has probably very slightly increased the performance of the example. Also, if support for optional UVs regressed then this example would break and I'm not aware of an alternative solution for end-to-end testing yet.

I don't feel strongly about though so I can change this and maybe raise a separate PR for adding Mesh::remove_attribute().

Reply:
Bevy examples are "minimal illustrations of best practices" first and everything else second. In this case, this is a minimal illustration of adding vertex colors to a mesh. Removing uvs won't always be necessary for supporting vertex colors, so I think doing it here detracts from the message (and for some readers might imply that removing them is necessary).

Reply:
Definitely a reasonable idea, but I think we should remove this line.